packages/main/src/Switch.ts

```diff
@@ -286,11 +286,12 @@ class Switch extends UI5Element implements IFormElement {
 
 		return {
 			main: {
-				"ui5-switch-desktop": isDesktop(),
+				"ui5-switch--desktop": isDesktop(),
 				"ui5-switch--disabled": this.disabled,
 				"ui5-switch--checked": this.checked,
 				"ui5-switch--semantic": this.graphical,
 				"ui5-switch--no-label": !hasLabel,
+				"ui5-switch--ios-device": isSafari(),
```

Comment:
Can we use ui5-switch--safari? Because people do use chrome on some ios devices, even if it sounds unnatural :D